test/integration/model/upsert.test.js

```diff
@@ -61,18 +61,18 @@ describe(Support.getTestDialectTeaser('Model'), () => {
     describe('upsert', () => {
       it('works with upsert on id', async function() {
         const created0 = await this.User.upsert({ id: 42, username: 'john' });
```

Comment:
Maybe destructure this for clarity?

Reply:
Agreed

```suggestion
        const [,created0] = await this.User.upsert({ id: 42, username: 'john' });
```

Then use `created0` instead of `created0[1]`

Reply:
Yeah, I didn't want to update all those test, but will do